### ISSUE TITLE
[Redesign] Restore Password Flow Test Case

### ISSUE THREAD

**Issue by Nodarianec** [ita-social-projects/Forum#937]
**Date Time**
18.11.2024
**Priority**
Medium
**Description**
Verify that all error messages are in red to attract attention and improve accessibility.
**Pre-requisites**
1.The website http://178.212.110.52/ is opened.
2.The user is registered.
Steps to be fulfilled before test:
1.the user is on the password recovery page http://178.212.110.52/password/reset/confirm/MTA2/cghiet-86f22906b66068223751c0170ceaf3a6

**Test Cases**
|      S#     |    Action   |  Test Data  | Expected Result | Actual Result |     P/F     |  Automated  |
|:-----------:|:-----------:|:-----------:|:---------------:|:-------------:|:-----------:|:-----------:|
|    7     |    Enter incorrect data in the password entry field.   |    <br>     |      Both error messages should be styled in red to draw attention and enhance accessibility.      |     Both error messages are formatted in red to attract attention and improve accessibility.     |    <passed    |    <br>     |
|    <br>     |    <br>     |    <br>     |      <br>       |     <br>      |    <br>     |    <br>     |
|    <br>     |    <br>     |    <br>     |      <br>       |     <br>      |    <br>     |    <br>     |
|    <br>     |    <br>     |    <br>     |      <br>       |     <br>      |    <br>     |    <br>     |
|    <br>     |    <br>     |    <br>     |      <br>       |     <br>      |    <br>     |    <br>     |

**Screenshots**
![image](https://github.com/user-attachments/assets/ecdda815-5a51-4b45-b5c2-67bd21851828)
**Environment:**

**Additional context**

 
 User story links E.g.: "User story #864 "

Labels to be added "Test case", Priority ("medium: "), Type ("UI, "Functional"),
